gpu_direct/gpu.c

```diff
@@ -2,7 +2,7 @@
  * SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
  */
 
-#include "gpu.h"
+#include "./gpu.h"
```

Comment:
why?

Reply:
This is a clang linter standard regarding including files - this forces to include the file in the same directory named `gpu.h` instead of looking for one in the global include list.